docs/7-infrastructure-configuration-management/7.1.4-terraform-providers.md

```diff
@@ -2,7 +2,7 @@
 docs/7-infrastructure-configuration-management/7.1.4-terraform-providers.md:
   category: Infrastructure as Code
   estReadingMinutes: 20
-  exercises:
+  exercises: 
```

Comment:
@mcbillings -- Please enable the VS code setting to remove trailing whitespace on save. You can do this by going to `Code > Settings` and searching for `files trim`. Then, check the `Trim Trailing Whitespace` option.